### PR TITLE
Dx 960 csr guide

### PR DESCRIPTION
**Pull request by dtolb**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

File: numbers/guides/csrLookup.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{{accountId}}/subscriptions/{{applicationID}}
+```
+
+{% endextendmethod %}
+
```
Comment (jmulford-bw) [inline numbers/guides/csrLookup.md]: Do we need this in here? We already have create subscriptions documented

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

File: numbers/guides/csrLookup.md (hunk 1 above)
Comment (dtolb) [reply numbers/guides/csrLookup.md]: Yes, as it's part of the holistic setup XP. Sadly still no UI for subscriptions. I updated with a disclaimer that you probably only need to do once.

Good call out!

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Review by jmulford-bw — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Review by jchavez443 — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-960-CSR-Guide.s3-website-us-east-1.amazonaws.com/

**Pull request merged**